docker-compose.yml

```diff
@@ -1,82 +1,8 @@
 version: "3.8"
 services:
 
-  db:
-    image: postgres:12-alpine
-    volumes:
-      - app-db-data:/var/lib/postgresql/data/pgdata
-    env_file:
-      - .env
-    environment:
-      - PGDATA=/var/lib/postgresql/data/pgdata
-
-  ml-worker:
-    image: 'docker.io/giskardai/ml-worker:${TAG-latest}'
-    hostname: giskard-embedded-docker-worker
-    build:
-      dockerfile: python-client/docker/Dockerfile
-      context: .
-    environment:
-      GIT_REPOSITORY: http://frontend/git/udf.git
-      GIT_USER: ${GIT_USER:-giskard}
-      GIT_PASSWORD: ${GIT_PASSWORD:-giskard123!}
-    volumes:
-      - giskard-home:/giskard-home
-
-  backend:
-    image: 'docker.io/giskardai/backend:${TAG-latest}'
+  mono:
+    image: 'docker.io/giskardai/giskard:${TAG-latest}'
```

Comment:
Do we really need to keep the docker compose since we just have a single image without anything special configured?

Reply:
I agree, we shouldn't mix monolith docker image with the compose. In case we start deploying giskard as SaaS on our own infra it'll make more sense to get back to using compose instead of a mono image

Reply:
Resolved